lib/SILOptimizer/Utils/ConstantFolding.cpp

```diff
@@ -1563,7 +1578,8 @@ void ConstantFolder::initializeWorklist(SILFunction &f) {
         continue;
       }
 
-      if (isApplyOfBuiltin(*inst, BuiltinValueKind::GlobalStringTablePointer)) {
+      if (isApplyOfBuiltin(I, BuiltinValueKind::GlobalStringTablePointer) ||
```

Comment:
Looks like this should be `*inst` instead of `I`. Did part of the change get dropped somewhere?